code/modules/projectiles/updated_projectiles/guns/smgs.dm

```diff
@@ -60,8 +60,8 @@
 
 /obj/item/weapon/gun/smg/m39/set_gun_config_values()
 	fire_delay = config.low_fire_delay
-	burst_delay = config.mlow_fire_delay
-	burst_amount = config.high_burst_value
+	burst_delay = config.min_fire_delay
+	burst_amount = config.med_burst_value
```

Comment:
This is an undocumented change.

Reply:
As above.

It's per the test merge; you merged the older iteration of this which had same values.